__tests__/Unit/pages/Tasks/index.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react';
+import Tasks from '@/pages/tasks';
+import { store } from '@/app/store';
+import { Provider } from 'react-redux';
+import { AppTreeType } from 'next/dist/shared/lib/utils';
+
+type TaskPageProps = {
+    dev: boolean;
+    pathname: string;
+    query: {
+        dev: string;
+    };
+    AppTree: AppTreeType;
+};
+
+jest.mock('next/router', () => ({
+    useRouter: () => ({
+        query: {
+            dev: 'true',
```

Comment:
why are we adding the feature flag here dev true if our intension of the PR is to remove it. Also why this new test. we must simply remove the flag from the older tests.

Reply:
It is not part of the feature flag it's a part of SSR. I keeping it because if remove this getInitialProps then the task page calls twice first one for the DEFAULT tab and second one for the SELECTED tab.